.github/workflows/license.yml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at:
+#
+#     https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+name: License
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  verifications:
+    name: Quickstart Verifications
```

Comment:
```suggestion
    name: License Check
```

Reply:
ah, sorry. I was just trying to set up GitHub action quickly and see if it's working or not. I'll make changes accordingly